src/bots/ama/subCommands/listEvents.js

```diff
@@ -10,50 +11,74 @@ const handler = async () => {
   if (events.length === 0) {
     await client.message.channel.send(getStrings().noEvents);
   } else {
-    client.message.channel.send(formatEvents(events));
+    let parsedCmd = parseCommandString();
+
+    client.message.channel.send(
+      formatEvents(events, parsedCmd.arguments.length > 0)
+    );
   }
 };
 
 /**
  * Formats events into a readable string
  * @param {Array.<Object.<string, any>>} events - list of event objects
+ * @param {boolean} showIds - trigger that shows event Database IDs if set to true
  * @returns {string} - message string with all events
  */
-const formatEvents = (events) => {
+const formatEvents = (events, showIds) => {
   let currentDate = new Date();
   let splitDate = partition(events, (e) => e.date > currentDate);
+  let eventString = '';
+
+  if (splitDate[1].length > 0) {
+    let pastEvents = '';
 
-  let upcomingEvent = getStrings(
-    `${splitDate[0][0].date.toDateString()}: ${splitDate[0][0].title}`
-  ).upcomingEvent;
+    for (let s in splitDate[1]) {
+      let next = splitDate[1][s];
 
-  let futureEvents = '';
+      pastEvents += `${
+        showIds ? next.id + ' ' : ''
+      }${next.date.toDateString()}: ${next.title}\n`;
+    }
 
-  for (let i = 1; i < splitDate[0].length; i++) {
-    let next = splitDate[0][i];
+    pastEvents = getStrings(pastEvents).pastEvents;
 
-    futureEvents += `${next.date.toDateString()}: ${next.title}\n`;
+    eventString += pastEvents + '\n';
   }
 
-  futureEvents = getStrings(futureEvents).futureEvents;
+  if (splitDate[0].length > 0) {
+    let upcomingEvent = getStrings(
+      `${
+        showIds ? splitDate[0][0].id + ' ' : ''
+      }${splitDate[0][0].date.toDateString()}: ${splitDate[0][0].title}`
+    ).upcomingEvent;
 
-  let pastEvents = '';
+    eventString += upcomingEvent + '\n';
 
-  for (let s in splitDate[1]) {
-    let next = splitDate[1][s];
+    let futureEvents = '';
 
-    pastEvents += `${next.date.toDateString()}: ${next.title}\n`;
-  }
+    for (let i = 1; i < splitDate[0].length; i++) {
+      let next = splitDate[0][i];
 
-  pastEvents = getStrings(pastEvents).pastEvents;
+      futureEvents += `${
+        showIds ? next.id + ' ' : ''
+      }${next.date.toDateString()}: ${next.title}\n`;
+    }
+
+    futureEvents = getStrings(futureEvents).futureEvents;
+
+    if (splitDate[0].length > 1) {
+      eventString += futureEvents + '\n';
+    }
+  }
 
-  return `${pastEvents}\n${upcomingEvent}\n${futureEvents}`;
+  return eventString;
 };
 
 const listEvents = {
-  example: 'list',
+  example: 'list [-i]',
```

Comment:
Just noting I really like this system of using unique IDs and think @jackwherry could use this to partially deal with duplicates in faq bot + for editing in the future

Reply:
I think he is using it. They're default IDs in mongo

Reply:
Ah haven't looked at his code yet, I generally don't look at draft PRs much :P

Reply:
I don't have any code that actually uses them yet, but that is how I'll do deletions and editing (same as the AMA bot). I think I'll just sort duplicates when you're using `++faq define` ascending by definition instead of using the IDs though.